Ports/AvailablePorts.md

```diff
@@ -198,6 +198,7 @@ This list is also available at [ports.serenityos.net](https://ports.serenityos.n
 | [`ruby`](ruby/)                                     | Ruby                                                            | 3.0.4                    | https://www.ruby-lang.org/                                                     |
 | [`sam`](sam/)                                       | Software Automatic Mouth (SAM)                                  | c86ea39                  | https://github.com/vidarh/SAM                                                  |
 | [`scummvm`](scummvm/)                               | ScummVM                                                         | 2.5.1                    | https://www.scummvm.org/                                                       |
+| [`sdl12-compat`](sdl12-compat/)                     | SDL2 compatibility layer for SDL 1.2 games                      | 1.2.52                   | https://github.com/libsdl-org/sdl12-compat/                                    |
```

Comment:
This should go right before `SDL2`, I think.

Reply:
And done!